packages/storybook/stories/ParticipantList/__snapshots__/ParticipantList.stories.storyshot

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
This is strange, why have all these unrelated storybook snapshots changed?

Reply:
Happens when you call a new mergeStyles in a *.styles.ts. Removed them now.